README.md

```diff
@@ -1,45 +1,36 @@
 # riff Sample: Node Square
 
-A sample function that multiplies a number by itself, returning the squared value.
+A sample JavaScript function that multiplies a number by itself, returning the squared value.
 
 ## Deployment
 
-1. Setup a running riff install (tested with riff 0.0.6)
+1.  Install riff.
 
-   See riff's [Getting Started](https://github.com/projectriff/riff/blob/master/Getting-Started.adoc) guide. Skip if you already have riff running.
+    These instructions assume riff v0.2.0 or later. See riff's [Getting Started](https://projectriff.io/docs/getting-started/) guide. Skip if you already have riff running.
 
-2. Install the node invoker
+2.  Create the function.
 
-   ```bash
-    riff invokers apply -f https://github.com/projectriff/node-function-invoker/raw/v0.0.8/node-invoker.yaml
-   ```
+    This assumes that the default namespace was initialized with credentials for your `DOCKER_ID` on [Docker Hub](https://docs.docker.com/docker-hub/).
 
-3. Clone this repo
+    ```bash
+    riff function create square \
+    --git-repo https://github.com/projectriff-samples/node-square  \
+    --artifact square.js \
```

Comment:
The advantage of using a package.json is that we don't need to specify the artifact. Your call as to which you prefer.

Reply:
I think a single file creates the least amount of friction
(users don't even need to know about npm - just the module.exports convention)